src/__tests__/__snapshots__/generate.test.ts.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`The \`generate\` command throws an error when you pass an invalid flag 1`] = `
+"Command failed: ./bin/run generate -t fake-component -n FakeComponent
+ [31m›[39m   Error: Expected --template=fake-component to be one of: react-component
+ [31m›[39m   See more help with --help
+"
+`;
```

Comment:
What does snapshot testing give us for tooling? I'm unsure of the value here.

Reply:
We used in Family Directed with to snapshot API error responses. So my thought was to do the same here, but I don't have a strong opinion so if you have a better idea, I'm all ears